plugins/source/googleanalytics/go.mod

```diff
@@ -129,6 +128,7 @@ require (
 	google.golang.org/genproto v0.0.0-20230920204549-e6e6cdab5c13 // indirect
 	google.golang.org/genproto/googleapis/api v0.0.0-20230920204549-e6e6cdab5c13 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231002182017-d307bd883b97 // indirect
+	google.golang.org/grpc v1.58.3 // indirect
```

Comment:
run `go mod tidy` as the `resources/plugin/client.go` clearly imports this

Reply:
Ran `go mod tidy` in https://github.com/cloudquery/cloudquery/pull/15171/commits/8e2bd94dc9fe6c7755a3ff9c604a6ecf383ac453